huaweicloud/resource_huaweicloud_rds_read_replica_instance.go

```diff
@@ -0,0 +1,390 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/golangsdk/openstack/rds/v3/instances"
+)
+
+func resourceRdsReadReplicaInstance() *schema.Resource {
+
+	return &schema.Resource{
+
+		Create: resourceRdsReadReplicaInstanceCreate,
+		Read:   resourceRdsReadReplicaInstanceRead,
+		Delete: resourceRdsReadReplicaInstanceDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"availability_zone": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"replica_of_id": {
```

Comment:
suggest to rename `replica_of_id` to `primary_instance_id`

Reply:
Ok, it seems more clearer.

Reply:
It has been renamed.